tests/phpunit/integration/Core/Authentication/Clients/Client_FactoryTest.php

```diff
@@ -57,4 +57,29 @@ public function test_create_client() {
 		$this->assertEquals( $token, $client->getAccessToken() );
 		$this->assertEquals( $login_hint, $client->getConfig( 'login_hint' ) );
 	}
+
+	/**
+	 * @dataProvider data_config_ip_resolve_values
+	 */
+	public function test_client_ip_resolve_config_options( $ip_resolve, $want ) {
```

Comment:
It's a minor point, but let's use `$expected_value` rather than `$want`, it would be more consistent with other tests.

I'll make the change myself to save the round trip.

Reply:
[Golang habits](https://go.dev/doc/tutorial/add-a-test) 😄